### PR TITLE
doc(function) add more php examples

### PR DESCRIPTION
**Pull request by lucasscw**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: compute/functions/reference-content/code-examples.mdx [hunk 1]
```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
...
+
+CRON jobs share the same pattern as HTTP calls. You can find the information passed into the CRON body in the event object:
+
 
-```
\ No newline at end of file
+<?php
+
+function handle($event, $context) {
+    var_dump($event);
```
Comment (thomas-tacquet) [inline compute/functions/reference-content/code-examples.mdx:956]: not necessary

File: compute/functions/reference-content/code-examples.mdx [hunk 2]
```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
...
+
+
+<?php
+
+function handle($event, $context) {
+    var_dump($event);
+    $queryparameter = (str)$event["queryStringParameters"]["parameter"];
+ 
+    $querybody = $event["body"];
+
```
Comment (thomas-tacquet) [inline compute/functions/reference-content/code-examples.mdx:940]: remove this line

File: compute/functions/reference-content/code-examples.mdx [hunk 3]
```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
...
+* `headers`: map(string)string - HTTP Request Headers
+* `method`: string - HTTP method used
+* `isBase64Encoded`: boolean - Whether the request body is base64 encoded.
+
+
+<?php
+
+function handle($event, $context) {
+    var_dump($event);
+    $queryparameter = (str)$event["queryStringParameters"]["parameter"];
```
Comment (thomas-tacquet) [inline compute/functions/reference-content/code-examples.mdx:937]: (str) not necessary

File: compute/functions/reference-content/code-examples.mdx [hunk 4]
```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
...
+* `body`: string|byte() - Body of the HTTP Request, you will have to parse it in your handler to use it properly.
+* `headers`: map(string)string - HTTP Request Headers
+* `method`: string - HTTP method used
+* `isBase64Encoded`: boolean - Whether the request body is base64 encoded.
+
+
+<?php
+
+function handle($event, $context) {
+    var_dump($event);
```
Comment (thomas-tacquet) [inline compute/functions/reference-content/code-examples.mdx:936]: not necessary

File: compute/functions/reference-content/code-examples.mdx [hunk 5]
```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
...
+    $cronbody = $event["body"];
+
+    return [
+	"body" => $cronbody,
+	"statusCode" => 200,
+	];
+}
```
Comment (thomas-tacquet) [inline compute/functions/reference-content/code-examples.mdx:963]: identation between snippets are different (tabs vs spaces), you should use a php formatter (some online tools are available)

File: compute/functions/reference-content/code-examples.mdx (hunk 2 above)
Comment (thomas-tacquet) [reply compute/functions/reference-content/code-examples.mdx:939]: the empty line 👌

File: compute/functions/reference-content/code-examples.mdx (hunk 5 above)
Comment (ldecarvalho-doc) [reply compute/functions/reference-content/code-examples.mdx:961]: @lucasscw is it ok to resolve this conversation and merge the PR?

File: compute/functions/reference-content/code-examples.mdx (hunk 2 above)
Comment (ldecarvalho-doc) [reply compute/functions/reference-content/code-examples.mdx:939]: @lucasscw is it ok to resolve this conversation and merge the PR?

**Review by bene2k1 — APPROVED** [review]

File: compute/functions/reference-content/code-examples.mdx [hunk 6]
```diff
@@ -896,5 +898,66 @@ function handle($event, $context) {
...
+    $queryparameter = (str)$event["queryStringParameters"]["parameter"];
+ 
+    $querybody = $event["body"];
+
+
+    return [
+	"body" => $querybody,
+	"statusCode" => 200,
+	];
+}
```
[the remark this answers is not in the document]
Comment (lucasscw) [reply compute/functions/reference-content/code-examples.mdx:945]: It works without

**Pull request merged**